bom-latest/pom.xml

```
@@ -17,7 +17,7 @@
         <structs-plugin.version>1.22</structs-plugin.version>
         <workflow-api-plugin.version>2.42</workflow-api-plugin.version>
         <workflow-cps-plugin.version>2.90</workflow-cps-plugin.version>
-        <workflow-job-plugin.version>2.40</workflow-job-plugin.version>
+        <workflow-job-plugin.version>2.41-rc1105.f84ef8f93137</workflow-job-plugin.version>
```

@car-roll 
```suggestion
        <workflow-job-plugin.version>2.41-rc1112.91a865190a8</workflow-job-plugin.version>
```

With incrementals fixed.